helix-core/src/main/java/org/apache/helix/manager/zk/ZkBucketDataAccessor.java

```diff
@@ -348,6 +356,14 @@ public void finalize() {
     close();
   }
 
+  int getAccessOption(long znodeTTLms) {
```

Comment:
private static?

Reply:
I need to access from tests as well, hence package-private. yes it can be made static

Reply:
changed